src/components/homepage-features.tsx

```diff
@@ -103,7 +102,32 @@ export default function HomepageFeatures(): JSX.Element {
       <div className={clsx('container--small', styles.footerCta)}>
         <h2>Ready to get started with Sequelize?</h2>
         <p>
-          Transactions, migrations, strong typing, JSON querying, and more.
+          <Link
+            className="topics"
+            to="/docs/v6/other-topics/transactions/"
+          > Transactions
+          </Link>,
+          <Link
+            className="topics"
+            to="/docs/v6/other-topics/migrations/"
+          > migrations
+          </Link>,
+          <Link
+            className="topics"
+            to="/docs/v6/other-topics/typescript/"
+          > strong typing
+          </Link>,
+          <Link
+            className="topics"
+            to="/docs/v6/other-topics/other-data-types/#json-sqlite-mysql-mariadb-and-postgresql-only"
+          > JSON querying
```

Comment:
Keep the space outside of the link, otherwise it looks weird 
![image](https://user-images.githubusercontent.com/1280915/166142766-48adbc2a-2328-49a4-9528-d7a1fd84725e.png)

Reply:
@fzn0x could you also fix this? I think we should be good to merge after that